games-fps/redeclipse/files/redeclipse

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd /usr/share/redeclipse
+[[ "$0" = "redeclipse_server" ]] && redeclipse_server_linux || redeclipse_linux
```

Comment:
Swap the double brackets for single brackets.

Reply:
don't do this. Gentoo explicitly recommends using bash `[[ ]]` over crappy POSIX `[ ]`: https://devmanual.gentoo.org/tools-reference/bash/index.html#single-versus-double-brackets-in-bash

Reply:
ignore my previous comment, I was just being delusional